17-react-native/classworks/Emelyanov_Egor/task/screens/CharactersList/CharactersList.screen.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```

Comment:
it's not a screen. it's a usual component. the name of file and place where it's located should be different